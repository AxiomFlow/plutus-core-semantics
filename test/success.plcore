(define result (success (lam x x)))
