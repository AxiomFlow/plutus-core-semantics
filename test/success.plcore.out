<T> <k> . </k> <env> result |-> ( success ( lam x x ) ) </env> </T>
